src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -612,4 +612,16 @@
   <data name="Convert_to_regular_constructor" xml:space="preserve">
     <value>Convert to regular constructor</value>
   </data>
+  <data name="Apply_0" xml:space="preserve">
+    <value>Apply '{0}'</value>
+  </data>
+  <data name="Apply_Copilot_suggestion" xml:space="preserve">
+    <value>Apply Copilot suggestion</value>
+  </data>
+  <data name="Apply_fix_from" xml:space="preserve">
+    <value>Apply fix from</value>
+  </data>
+  <data name="Warning_colon_AI_suggestions_might_be_inaccurate" xml:space="preserve">
+    <value>Warning: AI suggestions might be inaccurate.</value>
+  </data>
```

Comment:
none of these seem c# specific.

Reply:
True, but this feature is C# specific for now. All of this can be moved down in future.